recipes/spirv-headers/all/conanfile.py

```diff
@@ -1,6 +1,8 @@
 from conans import ConanFile, tools, CMake
 import os
+import glob
```

Comment:
```suggestion
```

useless now

Reply:
Well I was quite sure I removed that, that's why it's in the commit message. Anyway, fixed.